### PR TITLE
task/01 setup

### PR DESCRIPTION
**Pull request by SerhiiTrokhymets**
### Acceptance criteria for 1

- [x]  Mentor is added to pr
- [x]  AC copied to pr description
- [x]  link to deployed version added to pr
- [x]  About block (text and image) added to the main page
- [x]  On the widths 390px and 1280px, the site layout corresponds to the design

### Acceptance criteria for 1*

- [x]  Created separate component About
- [x]  Its styles added to [.module.css](https://github.com/css-modules/css-modules)
- [x]  For App.tsx styles moved to App.module.css

### CODE REVIEW

File: src/components/about/about.module.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    height: 294px;
+    border-radius: 30px;
+  }
+
+  .underline {
+    text-decoration: underline;
+  }
+}
+
+@media (min-width: 1280px) {
```
Comment (serhii-tsivinskyi-moc) [inline src/components/about/about.module.css:34]: Improve responsiveness between 390px and 1280px. Page should take whole available space of the screen.

File: src/components/about/AboutMe.component.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                <p>
+                    This page was developed during the course <span className={aboutStyles.underline}>&apos;Intro to React&apos;</span> from
+                    Masters Academy in 2024.
+                </p>
+                <p>
+                    This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own
+                    small project for the portfolio.
+                </p>
+                <p>
+                    You can contact me via <a href="https://www.facebook.com/profile.php?id=100011329003177">FaceBook</a> and check out my{' '}
```
Comment (serhii-tsivinskyi-moc) [inline src/components/about/AboutMe.component.tsx]: use `&nbsp;` for empty spaces

File: src/components/about/about.module.css (hunk 1 above)
Comment (SerhiiTrokhymets) [reply src/components/about/about.module.css:34]: done

File: src/components/about/AboutMe.component.tsx (hunk 2 above)
Comment (SerhiiTrokhymets) [reply src/components/about/AboutMe.component.tsx]: done

File: src/components/about/about.module.css (hunk 1 above)
Comment (serhii-tsivinskyi-moc) [reply src/components/about/about.module.css:34]: Screenshot 2024-04-27 at 13 00 11

Comment (serhii-tsivinskyi-moc) [reply src/components/about/about.module.css:34]: @SerhiiTrokhymets is it clear what do you need to do here?

Comment (SerhiiTrokhymets) [reply src/components/about/about.module.css:32]: @serhii-tsivinskyi-moc I get the idea, but I'm still figuring it out

Comment (SerhiiTrokhymets) [reply src/components/about/about.module.css:34]: fixed

Comment (serhii-tsivinskyi-moc) [reply src/components/about/about.module.css:34]: The image appears oversized below 1280px.

Comment (SerhiiTrokhymets) [reply src/components/about/about.module.css:34]: @serhii-tsivinskyi-moc 
I'm sorry, I don't understand what to do or what you mean

Comment (serhii-tsivinskyi-moc) [reply src/components/about/about.module.css:34]: Screenshot 2024-05-04 at 11 28 12

@SerhiiTrokhymets an image is huge comparing to a text

**Review by serhii-tsivinskyi-moc — APPROVED** [review]

**Pull request merged**